components/explore-samples-window-content.vue

```diff
@@ -12,15 +12,16 @@ const tooltip: Ref<HTMLElement | null> = ref(null);
 
 const { simpleItems } = useTEIHeaders();
 
-const filters: Array<"region" | "settlement"> = ["region", "settlement"];
+const filters: Array<"region" | "place"> = ["region", "place"];
 const ids = computed(() => {
 	return params.value.ids
 		? params.value.ids
 		: simpleItems.value
 				.filter((item) => params.value.dataType === item.dataType)
 				.filter((item) => {
-					if (!params.value.region && !params.value.settlement) return true;
+					if (!params.value.region && !params.value.place) return true;
 					const filter_results = filters.map((key) => {
+						//@ts-expect-error TODO: check if this is correct - key "place" does not exist on simpleteimetadata.place
```

Comment:
I commented this out during the last round of type fixes, but could you please take a look if this needs to be fixed upstream or here - thank you!